src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionOverviewViewController.swift

```diff
@@ -142,7 +142,7 @@ extension ExposureSubmissionOverviewViewController {
 // MARK: - ExposureSubmissionQRScannerDelegate methods.
 
 extension ExposureSubmissionOverviewViewController: ExposureSubmissionQRScannerDelegate {
-	func qrScanner(_ viewController: ExposureSubmissionQRScannerViewController, error: QRScannerError) {
+	func qrScanner(_ viewController: QRScannerViewController, error: QRScannerError) {
```

Comment:
Why?

Reply:
This makes it easier to test. QRScannerViewController is just a protocol that wraps the essential functionality of the ExposureSubmissionQRScannerViewController so it can be mocked in unit tests.